tests/python/unittest/test_runtime_extension.py

```diff
@@ -32,17 +34,18 @@ def _tvm_handle(self):
 
 
 def test_dltensor_compatible():
+    "Test DLTensor compatibility"
```

Comment:
nit: i think taht should be """ """-style comment, could you see if we're missing a pylint rule?

Reply:
It may be only in a newer version of pylint?
https://pylint.pycqa.org/en/latest/user_guide/messages/convention/bad-docstring-quotes.html